deploy/terraform/resources/variables.tf

```diff
@@ -190,6 +190,26 @@ variable "DEBUG_EMAIL" {
   default = "sanjay@developmentseed.org"
 }
 
+variable "RESOURCES_RG" {
```

Comment:
This is an extremely unnecessary change @szabozoltan69. We will have to revert all this. Sorry. 

@batpad

Reply:
We need these names visible as part of the templates because that's easy to debug. These resources are also managed manually and is a best practice to have these cleanly identified in the variables.tf. Any changes to these names should require a code update — that is intentional.